packages/tools/cookbook/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "@kadena/cookbook",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "scripts": {
+    "build": "",
+    "_phase:build": "",
+    "lint": "npx eslint ./mixins --ext .js,.ts --fix && npx eslint ./profile --ext .js,.ts --fix",
```

Comment:
These paths do not exist. We can do one of:

- add linting 
-  remove this

Reply:
I'll remove it 👍